src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1651,6 +1651,31 @@ GlobeControls.prototype.getScale = function getScale(pitch) {
 };
 
 /**
+ * To convert the projection in meters on the globe of a number of pixels of screen
+ * @param      {number} pixels count pixels to project
+ * @param      {number} pitch Screen pitch, in millimeters
+ * @return     {number} projection in meters on globe
+ */
+GlobeControls.prototype.pixelsToMeters = function pixelsToMeters(pixels, pitch) {
+    pitch = pitch || 0.28;
+    const scaled = this.getScale(pitch);
+    const size = pixels * pitch;
+    return size / scaled / 1000;
+};
+
+/**
+ * Projection on screen in pixels of length in meter on globe
+ * @param      {number}  value Length in meter on globe
+ * @param      {number}  pitch Screen pitch, in millimeters
+ * @return     {number}  projection in pixels on screen
+ */
+GlobeControls.prototype.metersTopixels = function metersTopixels(value, pitch) {
```

Comment:
While fixing the typo, you could also extract this feature to a specific commit.